TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/threading/MyLooperDelayedExecutor.java

```diff
@@ -0,0 +1,51 @@
+package com.novoda.tpbot.threading;
+
+import android.os.Handler;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MyLooperDelayedExecutor implements Executor.DelayedExecutor {
+
+    private final Handler handler;
+    private final Map<Action, Runnable> runnables;
+
+    public static MyLooperDelayedExecutor newInstance() {
+        Handler handler = new Handler();
```

Comment:
Should this executor specify the `Looper` too?

Reply:
Probably 🤔 I'll take a look 😄